Input/UserInput.cs

```diff
@@ -0,0 +1,9 @@
+namespace PhoneBookConsole.Input;
+
+public class UserInput
+{
+    public string GetInput()
+    {
+       return Console.ReadLine().Trim();
```

Comment:
It looks like we need a bit more validation. This is actually very important and will be necessary in most real life apps, it will be great practice! As you can see I can add an empty name, a phone number that's not a phone number and and e-mail that's not an e-mail. These shouldn't be accepted. :)
![image](https://user-images.githubusercontent.com/108711476/210950422-688276e8-b38d-4c04-aaa2-d658dc8ded14.png)

Reply:
done